roles/cleanup/tasks/logging.yml

```diff
@@ -1,10 +1,19 @@
-- name: keystone logging.conf
-  file: dest=/etc/keystone/logging.conf state=absent
-- file: dest=/var/log/keystone/error.log state=absent
-- file: dest=/var/log/keystone/access.log state=absent
+- name: remove old log configs / files
+  file: dest={{ item }} state=absent
+  with_items:
+    - /etc/nova/logging.conf
+    - /etc/cinder/logging.conf
+    - /etc/keystone/logging.conf
```

Comment:
Hmm, there are no checks in place. Probably something that could be added in the future too, though all of the logging templates like this were removed back when we first ran into the Python <2.7.4 syslog handling issue.